latex/script/mattex1.py

```diff
@@ -1,15 +1,6 @@
-
-
 import matplotlib.pyplot as plt
 import numpy as np
 
-
-
-
-
-
-
-
 x = np.linspace(0, 10, 1000)
 y = x ** np.sin(x)
 
```

Comment:
May consider the layout on the tex slides for this, slide 162 ff. We used this spaced layout to easier introduce the latter matplotlib settings without changing the position of "reference" lines.